modules/deploy/pages/deployment-option/cloud/managed-connectors/create-mmaker-checkpoint-connector.adoc

```diff
@@ -11,6 +11,12 @@ from other Kafka clusters.
 * A service account with read-only access to the external cluster is available.
 * The Kafka cluster topics connector is running for the same source cluster, with a matching configuration.
 
+== Limitations
+
+The MirrorMaker2 Checkpoint connector migrates (recently) active consumer groups. It does not migrate
```

Comment:
Having trouble what you mean here with the "recently" parenthetical? The docs don't specify that something was recently added--we either support the documented functionality or do not.

I've tried to rewrite this blurb a few times, but realized that it is simply unclear and I am not sure what you are trying to say. Can you try again? Sorry--want this to be both clear and accurate.